org.blackcoinmore.blackmore-qt.metainfo.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.blackcoinmore.blackmore-qt</id>
+  <name>Blackcoin More</name>
+  <project_license>MIT</project_license>
+  <metadata_license>CC0-1.0</metadata_license>
+  <summary>
+    Fully validating Blackcoin peer-to-peer proof-of-stake network node, wallet and GUI
+  </summary>
+  <url type="homepage">
+    https://blackcoinmore.org/
+  </url>
+  <url type="bugtracker">
+    https://github.com/coinblack/blackcoin-more/issues/new?title=[flatpak]
+  </url>
+  <launchable type="desktop-id">
+    org.blackcoinmore.blackmore-qt.desktop
+  </launchable>
+  <description>
+    <p>
+    Blackcoin is the original decentralised and eco-friendly currency with near-instant transaction speeds and negligible transaction fees built upon Proof of Stake 3.0 since 2014.
+    </p>
+  </description>
+  <screenshots>
+    <screenshot>
+      <image type="source">
+        https://raw.githubusercontent.com/coinblack/blackmore-flatpak/main/wallet.png
+      </image>
+    </screenshot>
+  </screenshots>
+  <releases>
+    <release date="2023-07-04" version="13.2.1"/>
+  </releases>
+  <content_rating type="oars-1.1"/>
```

Comment:
I don't think this `content_rating` is appropriate for this app.

Reply:
I copied this from https://github.com/flathub/org.bitcoincore.bitcoin-qt/blob/master/org.bitcoincore.bitcoin-qt.metainfo.xml#L48

Should I remove it?

Reply:
it's wrong were you copied it from.

This app deal with "money"

Reply:
https://github.com/flathub/flathub/pull/4389/commits/c0273e03b17ee8923d7f09e35a695e246071e88a

change to 1.0